README.md

```diff
@@ -87,60 +89,61 @@ steps:
+| `mvn_executable_path` | Specifies absolute path of `mvn` binary. | `mvn` 
+| `gradle_executable_path` | Specifies absolute path of `gradle` binary. | `gradle`
+| `npm_executable_path` | Specifies absolute path of `npm` binary. | `npm`
+| `go_executable_path` | Specifies absolute path of `go` binary. | `go`
+| `python3_executable_path` | Specifies absolute path of `python3` binary, `python3` takes precedence over `python`. | `python3`
+| `pip3_executable_path` | Specifies absolute path of `pip3` binary, `pip3` takes precedence over `pip`. | `pip3`
+| `python_executable_path` | Specifies absolute path of `python` binary, `python3` takes precedence over `python`. | `python`
+| `pip_executable_path` | Specifies absolute path of `pip` binary, `pip3` takes precedence over `pip`. | `pip`
+| `syft_executable_path` | Specifies absolute path of `syft` binary. | `syft`
+| `syft_config_path` | Specifies absolute path to the Syft configuration file. | 
+| `skopeo_executable_path` | Specifies absolute path of `skopeo` binary. | `skopeo`
+| `skopeo_config_path` | Specifies absolute path to the authentication file used by `skopeo inspect`. | 
+| `docker_executable_path` | Specifies absolute path of `docker` binary. | `docker`
+| `podman_executable_path` | Specifies absolute path of `podman` binary. | `podman`
+| `image_platform` | Specifies platform used for multi-arch images. |
 
 ## Action Outputs
 
-- **rhda_report_json**: Path to generated Red Hat Dependency Analytics Report in JSON format.
-- **rhda_report_sarif**: Path to generated Red Hat Dependency Analytics Report in SARIF format.
-- **artifact_id**: ID of the uploaded artifact.
+- **rhda_report_json**: Path to the generated Red Hat Dependency Analytics report in JSON format.
+- **rhda_report_sarif**: Path to the generated Red Hat Dependency Analytics report in SARIF format.
+- **artifact_id**: The identifier of the uploaded artifact.
 
 ## Scanning Pull Requests
 
-This action can run RHDA scans on pull requests. Because the action must check out the pull request's code in order to scan it, the [`pull_request_target` trigger](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) must be used. <!-- markdown-link-check-disable-line -->
-
-After the RHDA scan is approved and the workflow runs, a label indicating the scan result will be added to the pull request.
+<br >This action can also run RHDA scans on pull requests.
+Because the action must check out the pull request's code to scan it, you must use a [`pull_request_target` trigger](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target).
 
-The following snapshot shows vulnerability details in the GitHub UI for a pull request.
+<br >After approving the RHDA scan, the workflow runs, and labels the scanning results to the pull request.
```

Comment:
Sorry, the approval process has been removed eventually (i forgot to remove its mention from the documentation). now its just "After the RHDA scan workflow runs, it labels the scanning results to the pull request."

Reply:
Okay, I'll update the content.